### DISCUSSION TITLE
Using NFC Tags With Homebox

### DISCUSSION THREAD

**Discussion by danielrosehill** [hay-kot/homebox#856] [Show and tell]
Nothing too fancy about this but thought I'd share anyway.

Here's a HDMI-HDMI cable that I wanted to add into my inventory but .. getting a QR code on this would be very tricky!

![Inventory20240319_032224](https://github.com/hay-kot/homebox/assets/40233203/8a998f50-fe56-4a27-869e-e82b7273f96d)

So I bought a very pack of very small NFC tags. They fit into just the connector end!

![Inventory20240319_032526](https://github.com/hay-kot/homebox/assets/40233203/cc8aa74f-7cc7-4028-abe6-8a7993a2bfff)

Writing to them is easy.

I firstly created an item in Homebox (I'm using an 'Added NFC Tag' label to categorise these):

![hb entry](https://github.com/hay-kot/homebox/assets/40233203/1a3a9f95-a721-4fca-8775-18e8b565109a)

In NFC Tools on Android just create a custom URL record with the Homebox asset ID:

![Screenshot_20240319_033259_NFC Tools](https://github.com/hay-kot/homebox/assets/40233203/281f872f-acfc-4854-b47f-6f0b4b4ea99a)

Then approach the tag and write it!

![Screenshot_20240319_033313_NFC Tools](https://github.com/hay-kot/homebox/assets/40233203/eab1fd4e-f137-49f8-afc8-9831a61ad9f8)

You can even use the 'lock tag' command to prevent them from being overwritten (note: this action is irreversible):

![Screenshot_20240319_033704_NFC Tools](https://github.com/hay-kot/homebox/assets/40233203/3712d66e-8daf-4147-97c9-8819f3fbe3c2)

**Comment by markvdb** [comment]
Very inspiring. Thank you.
